tests/filecheck/backend/csl/print_csl.mlir

```diff
@@ -626,12 +628,12 @@ csl.func @builtins() {
 // CHECK-NEXT:   @set_rectangle(x_dim, y_dim);
 // CHECK-NEXT:   const x_coord0 : i32 = 0;
 // CHECK-NEXT:   const y_coord : i32 = 0;
-// CHECK-NEXT:   @set_tile_code(x_coord0, y_coord, "file.csl", )
+// CHECK-NEXT:   @set_tile_code(x_coord0, y_coord, "file.csl", );
```

Comment:
Is this trailing comma in the call wanted / correct?

Reply:
Correct yes, wanted not really.